src/components/atoms/Iframe/Iframe.test.tsx

```diff
@@ -75,9 +84,11 @@ describe('Iframe - Snapshots', () => {
       <Iframe
         src="https://example.com/embed"
         title="Example embed"
-        width="100%"
-        height="315"
-        data-testid="test-iframe"
+        className="custom-class"
+        width="640"
```

Comment:
The `width=100%` and `height=315` set previously was done so to match the implementation.